tools/cell_census_builder/globals.py

```diff
@@ -220,32 +228,22 @@
 # Global TileDB context
 _TileDB_Ctx: tiledb.Ctx = None
 
-# The logical timestamp at which all builder data should be recorded
-WRITE_TIMESTAMP = int(time.time() * 1000)
-
-# Using "end of time" for read_timestamp means that all writes are visible, no matter what write timestamp was used
-END_OF_TIME = 0xFFFFFFFFFFFFFFFF
-
 
 def SOMA_TileDB_Context() -> soma.options.SOMATileDBContext:
     global _SOMA_TileDB_Context
     if _SOMA_TileDB_Context is None or _SOMA_TileDB_Context != TileDB_Ctx():
```

Comment:
To fix a bug I introduced on an earlier PR:
```suggestion
    if _SOMA_TileDB_Context is None or _SOMA_TileDB_Context.tiledb_ctx != TileDB_Ctx():
```

Reply:
But maybe it's time to just eliminate support for custom `tiledb_ctx` values (remove the setter), since we don't use it, allowing us to remove this check.

Reply:
Yeah, for now, I'll just make it a cached value.  Good call.